src/main/java/com/desiato/puresynth/exceptions/InvalidTokenException.java

```diff
@@ -0,0 +1,11 @@
+package com.desiato.puresynth.exceptions;
+
+public class InvalidTokenException extends RuntimeException {
```

Comment:
I think it could be an explicit `Exception`.

Reply:
By that, do you mean extending Exception?